IoT_master/tools/camera_initialize.py

```diff
@@ -0,0 +1,37 @@
+import cv2
+
+class Camera:
+    def __init__(self):
+        self.login = 'admin'
+        self.password = 'qwerty123456'
+        self.ip_address = '192.168.1.108'
+        self.channel = '1'
+        self.adres_part_1 = 'rtsp://'+self.login+':'+self.password+'@'+self.ip_address+'/cam/realmonitor?channel='
```

Comment:
Does it really have to be specified here?
Move it maybe to separate static method that will generate appropriate string based on arguments passed.

Also, I see that probably this line is too long to pass tests.
Have you run `make test` command on your code?

Reply:
I have run tox test if that's what you mean.